source/support.tex

```diff
@@ -3683,7 +3683,7 @@
 \begin{note}
 If
 \tcode{rethrow_exception} rethrows the same exception object (rather than a copy),
-concurrent access to that rethrown exception object may introduce a data race.
+concurrent access to that rethrown exception object could introduce a data race.
```

Comment:
```suggestion
concurrent access to that rethrown exception object can introduce a data race.
```

Reply:
Really, this is a possibility for the user of the document? I thought data races are something that are imposed onto the user by happenstance.
anyway, fixed as suggested